manifests/params.pp

```diff
@@ -22,5 +22,5 @@
   $user = 'qpidd'
   $group = 'qpidd'
 
-  $server_packages = ['qpid-cpp-server', 'qpid-cpp-client', 'python-qpid-qmf', 'python-qpid', 'policycoreutils-python', 'qpid-tools']
+  $server_packages = ['qpid-cpp-server', 'qpid-cpp-client', 'python-qpid-qmf', 'python-qpid', 'policycoreutils-python', 'qpid-tools', 'qpid-cpp-server-linearstore']
```

Comment:
Is it still in EPEL? I think it makes sense, this should probably be parameter and give the option of providing a store or not (same for things like installing the client, tools, etc).